ivy_tests/test_ivy/test_frontends/test_numpy/test_linear_algebra/test_norms_and_other_numbers.py

```diff
@@ -1,6 +1,8 @@
 # global
 from hypothesis import strategies as st, assume
 
+import ivy
```

Comment:
No need to import ivy.

Reply:
i use ivy later on in the file on line 241 when requiring the p parameter to be `ivy.inf` or `-ivy.inf` so removing it will cause an error on that line